src/nnet3/nnet-simple-component.cc

```diff
@@ -119,16 +129,27 @@ void DropoutComponent::Propagate(const ComponentPrecomputedIndexes *indexes,
 
   BaseFloat dropout = dropout_proportion_;
   KALDI_ASSERT(dropout >= 0.0 && dropout <= 1.0);
+  if(dropout_per_frame_)
```

Comment:
Please use the correct code style.  Should be
```
   if (x) { 
 ...
```
and note the space after if.  You can run misc/maintenance/cpplint.py on your code to check for style problems.